File: Proyectos_python/Tablas de verdad.py
print("Tabla de verdad para AND:")
print("A\tB\tA AND B")
for a in [False, True]:
    for b in [False, True]:
        print(f"{a}\t{b}\t{a and b}")

print("\nTabla de verdad para OR:")
print("A\tB\tA OR B")
for a in [False, True]:
    for b in [False, True]:
        print(f"{a}\t{b}\t{a or b}")

print("\nTabla de verdad para NOT:")
print("A\tNOT A")
for a in [False, True]:
    print(f"{a}\t{not a}")

File: Proyectos_c/Tablas_de_verdad.c
#include <stdio.h>

int main() {
    int a, b;

    printf("Tabla de verdad de AND\n");
    printf("a\tb\ta AND b\n");
    for (a = 0; a <= 1; a++) {
        for (b = 0; b <= 1; b++) {
            printf("%d\t%d\t%d\n", a, b, a && b);
        }
    }

    printf("\nTabla de verdad de OR\n");
    printf("a\tb\ta OR b\n");
    for (a = 0; a <= 1; a++) {
        for (b = 0; b <= 1; b++) {
            printf("%d\t%d\t%d\n", a, b, a || b);
        }
    }

    printf("\nTabla de verdad de NOT\n");
    printf("a\tnot a\n");
    for (a = 0; a <= 1; a++) {
        printf("%d\t%d\n", a, !a);
    }

    return 0;
}